taxiunn/driver_auth/views.py

```diff
@@ -0,0 +1,92 @@
+from smtplib import SMTPException
+
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from .serializers import (
+    DriverRegisterSerializer,
+    DriverVerifyRegisterSerializer,
+)
+
+from taxiunn.verification import (
+    RegistrationCache,
+    make_verification_code,
+    send_verification_code,
+)
+
+
+class RegistrationView(APIView):
+    """View регистрации водителя."""
+
+    def post(self, request) -> Response:
+        serializer = DriverRegisterSerializer(data=request.data)
+
+        if not serializer.is_valid():
+            return Response(
+                serializer.errors,
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        email = serializer.validated_data['email']
+        verification_code = make_verification_code()
+
+        RegistrationCache.save(
+            email=email,
+            code=verification_code,
+            data=serializer.validated_data,
+        )
+
+        try:
+            send_verification_code(
+                email=email,
+                verification_code=verification_code,
+            )
+        except SMTPException:
+            return Response(
+                {'message': 'Email not found!'},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        return Response(
+            {'message': "You\'ve successfully registered!"},
```

Comment:
`"You\'ve successfully registered!"` -> `print` -> `You've successfully registered!`
`"You've successfully registered!"` -> `print` -> `You've successfully registered!`